experiments/zfs-localpv/functional/backup_and_restore/test.yml

```diff
@@ -124,6 +124,14 @@
 
           when: action == 'restore'
 
+        - block:
+
+          - name: Restore the incremental backups
+            include_tasks: "./incr_restore.yml"
+            
+          when: action == 'incr_restore'
```

Comment:
@w3aman give the file name with meaning incremental_backup.  and for action also